todoman/confspec.ini

```diff
@@ -14,6 +14,13 @@ color = option('always', 'auto', 'never', default='auto')
 # this option is not specified the system locale's is used.
 date_format = string(default='%x')
 
+# The date format used both for displaying times, and parsing input times.
+# Default is `%H:%M`.
+time_format = string(default='%H:%M')
+
+# The string used to separate date and time when displaying and parsing.
+dt_separator = string(default=' ')
```

Comment:
As a default, I'd prefer to have:
```python
if time_format:
    return ' '
else:
    return ''
```

This way, we avoid an extra space when there's no `time_format` (I think it's burdensome for users to have to _disable_ the separator when they don't use times.